README.md

```diff
@@ -278,20 +272,14 @@ through the gate, either because:
 * they're on the list
 * they're part of the first _sample_ users
 
-Sampling is done by checking the character with which _user_ begins. For example,
-if _sample_ is `0.5` and the gate uses the default
-[_sampleCharacterSet_](#newusergateencodedgate-options), _user_ would be allowed
-through the gate if it began with any character between `a-n` (halfway through
-the alphabet).
-
-Users are unlikely to be uniformly distributed over _sampleCharacterSet_ and
-thus a _sample_ of `0.5` won't exactly select for 50% of users. However, this
-sampling technique is deterministic: a user will either always be allowed through
-the gate, even if they reload, or they never will.
+Sampling is done by hashing the _user_ string and projecting it onto a sample
+space - this is both deterministic and uniformly random.
 
 Checking against the list requires an exact match. However, sampling is
 case-insensitive.
 
+**In v3.x**, this returns `true` if _user_ is allowed through the gate, `false` otherwise.
```

Comment:
I would just edit the below line to say "In v2.x and above", then we don't have to keep adding bits here with new versions :)

Reply:
https://github.com/mixmaxhq/user-gate/pull/5/commits/c5758c56ac810bc2afc0cb0458e5b1b2c2bd80e7